src/T4Generator/Program.cs

```diff
@@ -359,4 +362,26 @@ static void ProcessTemplate(string[] args)
             }
         }
     }
+
+	static class Tools

```

Comment:
That will do! Can you move the class to another file, and call it `StringHelper` maybe ?

Reply:
Which namespace? Or, class Tools, without namespace?

Reply:
Same namespace, another file.